### DISCUSSION TITLE
Send OSC from MA3, receiving with library

### DISCUSSION THREAD

**Discussion by SkowroFl** [sstaub/gma3#4] [Q&A]
Hey Stefan,

thank you very much for this nice library. I am using it with an arduino mega and w5500 shield. Maybe just a simple question: What is the command in the MA3 when I want to send e.g. "Next page" from MA3 to my arduino over OSC? I want that the Arduino knows when I am changing faderpage.

Thanks a lot.

**Comment by sstaub** [comment]
Have a look to the manual https://help2.malighting.com/Page/grandMA3/keyword_sendosc/en/1.6/osc

**Reply by sstaub** [reply]
Your personal reply message must start with the configured Prefix, e.g. `/gma3/`, so it should possible to use the UDP parser described in the Readme.
